css/index.css

```diff
@@ -1675,3 +1675,14 @@ Logic Interface CSS
     opacity: 0.3;
     pointer-events: none;
 }
+
+#recordPointCloudsButton {
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    left: 20px;
+    top: calc(100vh - 20px - 40px);
```

Comment:
I believe this can just be `bottom: 20px` when using position:absolute

Reply:
thanks, updated!